terraso_backend/apps/graphql/schema/users.py

```diff
@@ -35,19 +36,30 @@
 logger = structlog.get_logger(__name__)
 
 
-class UserNode(DjangoObjectType):
-    id = graphene.ID(source="pk", required=True)
+class UserFilter(FilterSet):
+    user__in_project = CharFilter(method="filter_user_in_project")
```

Comment:
suggestion: i used this pattern in the sites filter, it's more concise and will automagically put the right type!
```suggestion
    user__in_project = TypedFilter(field_name="collaboration_memberships__membership_list__project")
```

Reply:
Nice! I missed this, this is awesome.